web/.prettierrc.yaml

```diff
@@ -0,0 +1,6 @@
+printWidth: 75
```

Comment:
Personal preference on `printWidth` - I'd go with 100 for a typescript project since the types add additional horizontal width.